webplugin/lib/js/dom-purify-3.1.4.min.js

```diff
@@ -0,0 +1,684 @@
+/*! @license DOMPurify 3.1.4 | (c) Cure53 and other contributors | Released under the Apache license 2.0 and Mozilla Public License 2.0 | github.com/cure53/DOMPurify/blob/3.1.4/LICENSE */
```

Comment:
this file doesn't look like a minified version, please recheck

Reply:
ok, it looks like I have done the formatting here  😄
will revert this